### ISSUE TITLE
Code Issue in RomLDR 

### ISSUE THREAD

**Issue by kiwisincebirth** [wwarthen/RomWBW#465]
While browsing code i found what looks like issue in RomLDR.asm (below from list file)

```
setcon: ...

0909   84A7 7A          	ld	a,d			; mask off current
0910   84A8 E6 00       	and	$11100000		; baud rate
0911   84AA 21 A6 8F    	ld	hl,newspeed		; and load in new
0912   84AD B6          	or	(hl)			; baud rate
0913   84AE 57          	ld	d,a
```

The AND on line 910 is anding using $ hex value when it should be a B binary, this is confirmed by the generated code E6 00, when should be E6 E0 - consequence of this is the MSB 3 bits are being lost.

**Comment by wwarthen** [comment]
Yikes.  Thank you very much @kiwisincebirth.  Good catch.  That would have cost me a couple hours of debugging someday!

Thanks, Wayne

**Comment by wwarthen** [comment]
Corrected in commit a8f2aa7
